gm4_shapeless_portals/data/gm4/advancements/root.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
this advancement isn't necessary since this module doesn't add any visual advancements